### PR TITLE
Use an explicit wait for the modal dialog

### PR DESCRIPTION
**Pull request by brunelli**
When using an implicit wait we need to wait a predefined amount of time (10 seconds) to close the popup. Instead, we can use an explicit wait to detect the presence of the element and save more than 9 seconds.

### CODE REVIEW

File: MinervaBot.py [hunk 1]
```diff
@@ -33,10 +36,16 @@
...
 # Acha o botao de fechar o popup e o fecha
-driver.implicitly_wait(10)  # Aguarda o modal dialog estar disponivel
-popup = driver.find_element_by_css_selector(
-    '.modal-footer > button[data-dismiss="modal"]')
-popup.send_keys(Keys.RETURN)
+try:
+    WebDriverWait(driver, 10).until(
+        EC.presence_of_element_located((By.ID, 'myModal'))
+    ) # Aguarda o popup estar disponivel
+except:
```
Comment (mvinoba) [inline MinervaBot.py]: It's better to do a `except TimeoutException` than to just use a bare `except`.

I guess we can also do some error handling, at least logging it and/or printing out should be fine. How do you feel about that?

**Review by mvinoba — COMMENTED** [review]
Thanks! This is much better than implicitly waiting

File: MinervaBot.py (hunk 1 above)
Comment (brunelli) [reply MinervaBot.py]: Good catch! From my understanding, a `TimeoutException` will only be raised when/if they remove the popup or change its ID, that's why I'm `pass`ing the exception.

---

About error handling, I'm planning another PR to deal with 1) unsuccessful logins and 2) no borrowed books.

In the first case maybe we should fallback to inputting the login and password via CLI. I'm also trying to think of a better way to store login info.

In the second case, I believe doing nothing or just printing a message would be the better choice.

Comment (mvinoba) [reply MinervaBot.py]: LGTM, I'm also interested in improving the login information processing. Thanks!

**Pull request merged**